x-pack/plugins/observability_solution/dataset_quality/public/components/flyout/flyout_summary/flyout_summary.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import { OnRefreshProps, OnTimeChangeProps, EuiSpacer } from '@elastic/eui';
+
+import { DegradedDocs } from '../degraded_docs_trend/degraded_docs';
+import { DataStreamDetails } from '../../../../common/api_types';
+import { DEFAULT_TIME_RANGE } from '../../../../common/constants';
+import { useDatasetQualityContext } from '../../dataset_quality/context';
+import { FlyoutDataset, TimeRangeConfig } from '../../../state_machines/dataset_quality_controller';
+import { FlyoutSummaryHeader } from './flyout_summary_header';
+import { FlyoutSummaryKpis, FlyoutSummaryKpisLoading } from './flyout_summary_kpis';
+
+const DEFAULT_REFRESH = { value: 60000, pause: false };
```

Comment:
Could we maybe share this default in `common/constants.ts`? we are also using it for the table